cli/main.go

```diff
@@ -324,7 +324,7 @@ func exit(ctx string, err error, ctype string) {
 
 	if errors.Is(err, api.ErrNotImplemented) {
 		name := metrics.GetCommand(os.Args[1:])
-		fmt.Fprintf(os.Stderr, "Command %q not available in current context (%s)\n", name, ctx)
+		fmt.Fprintf(os.Stderr, "Command %q not available in current context (%s). %q\n", name, ctx, err)
```

Comment:
Did you mean
```suggestion
		fmt.Fprintf(os.Stderr, "Command %q not available in current context (%s). %v\n", name, ctx, err)
```
maybe? It’s kind of unusual to use ` %q `  for errors, I think 🤔

Reply:
`%v` would show a Golang formatted string. As this is user facing we just want the error string

Reply:
An error is a string, really 😁 For formatting both `%v` and `%s` for an error do the same: https://play.golang.org/p/MesffycUwLV

Reply:
Maybe this should just be
```suggestion
		fmt.Fprintf(os.Stderr, "Command %q not available in current context (%s): %s\n", name, ctx, err)
```
?